### PR TITLE
Made the changes In vendor Dashboard

### LINKED ISSUE #107: Complete the vendor dashboard
**Is your feature request related to a problem? Please describe.**
Add all the services available to a vendor

### PR DESCRIPTION
**Pull request by ritwizsinha**
Fixes #107 

Changes:
1.Added the vendor dashboard

Surge Deployment Link: https://pr-117-sahaayak.surge.sh

Screenshot from 2019-05-24 19-46-30

### CODE REVIEW

File: src/components/VendorDashboard/VendorDashboard.js [hunk 1]
```diff
@@ -1,14 +1,125 @@
...
+  };
+  getVendorServices = (id) => {
+    this.props.getServices(id);
+  }
+  componentDidMount() {
+    const { id } = this.props.user;
+    this.getVendorServices(id);
+  }
+
+  enterLoading = () => {
```
Comment (harsh-jindal) [inline src/components/VendorDashboard/VendorDashboard.js]: Where is this been used

File: src/components/VendorDashboard/VendorDashboard.js [hunk 2]
```diff
@@ -1,14 +1,125 @@
...
+    this.setState({ loading: true });
+  };
+
+  enterIconLoading = () => {
```
Comment (harsh-jindal) [inline src/components/VendorDashboard/VendorDashboard.js]: Remove this function. Not required.

**Review by harsh-jindal — CHANGES_REQUESTED** [review]

**Comment by harsh-jindal** [comment]
@ritwizsinha Make the above changes. 
And don't open new PR for the same issue. It is tough to review and it is not a good practice

**Comment by ritwizsinha** [comment]
Changes done @harsh-jindal

**Review by harsh-jindal — APPROVED** [review]

**Pull request merged**